x-pack/plugins/session_view/public/components/tty_player/hooks.ts

```diff
@@ -97,16 +91,13 @@ export const useIOLines = (pages: ProcessEventsPage[] | undefined) => {
               const splitLines = process.io.text.split(TTY_LINE_SPLITTER_REGEX);
               const combinedLines = [splitLines[0]];
               const previousProcessId = previous[previous.length - 1]?.event.process?.entity_id;
-              const currentProcessLineInfo: ProcessEntityIdIOLine = {
-                value: entityIdLineMap[process.entity_id]?.value ?? previous.length,
-              };
 
-              if (previousProcessId && previousProcessId !== process.entity_id) {
-                entityIdLineMap[previousProcessId].next = currentProcessLineInfo.value;
-                currentProcessLineInfo.previous = entityIdLineMap[previousProcessId].value;
-              }
               if (previousProcessId !== process.entity_id) {
-                entityIdLineMap[process.entity_id] = currentProcessLineInfo;
+                const processLineInfo: ProcessStartMarker = {
+                  line: previous.length || 0,
```

Comment:
why not just return `line: previous.length`? Is there a case where `previous.length`  is less than 0 or undefined?